firmware/controllers/can/can_dash.cpp

```diff
@@ -1243,6 +1243,20 @@ static void populateFrame(Aim5f7& msg) {
 	msg.LambdaTarget1 = engine->fuelComputer->targetLambda;
 	msg.LambdaTarget2 = engine->fuelComputer->targetLambda;
 }
+// User Specific frames for EcuMaster PMU
```

Comment:
I am overall OK with the change but please improve code comments:

please clarify what packets follow what spec what version. maybe even a URL of spec into the source code

EcuMaster reference is super confusing. Are those packets totally random user-defined format? if that's just random format Lua is the proper way to go. if that's totally random custom format why does it matter who is the consumer EcuMaster or not?